### PR TITLE
S2t4 readme

### PR DESCRIPTION
**Pull request by AlisonYoon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
 # 2019-top-cat
 
+#### API scrape
```
Comment (mallowmew) [inline README.md]: Turn this into a second level heading? `##`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
 # 2019-top-cat
 
+#### API scrape 
+
+###### *APIScrape/scrape.php*
```
Comment (mallowmew) [inline README.md]: Turn this into a third level heading? `###`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,35 @@
...
+* Send API request to CatAPI using cURL
+
+* Get response from the CatAPI
+    * List of cat breeds
+    * All the img src url of all the cat breeds
+    
+* Fill the DB tables ( breed, img ) with the sanitized data from the process above
+
+* To run it, in the command line type 
+**(you must be inside the APIScrape directory)**
```
Comment (mallowmew) [inline README.md]: This is a really important line but it looks optional (in parentheses).

It's not really important how the scrape works - you can read the code to find that out. The readme reader is interested in how to use it.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,35 @@
...
+
+    ``` php scrape.php ```
```
Comment (mallowmew) [inline README.md]: Lead line with a $ to help show that it's a command line instruction.

File: README.md (hunk 1 above)
Comment (AlisonYoon) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (AlisonYoon) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (AlisonYoon) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (AlisonYoon) [reply README.md]: ok

**Review by mallowmew — APPROVED** [review]

**Pull request merged**